script/test/fixtures/multiple-compose-files/docker-compose.yml

```diff
@@ -0,0 +1,10 @@
+web:
+  image: tuna/docker-counter23
+  ports:
+    - "5000:5000"
+  links:
+    - redis
+redis:
+  image: redis:3.0
+  ports:
+    - "6379"
```

Comment:
things as in same parameters like env vars from 1st docker-compose file get over-written in second!